### ISSUE TITLE
VSCode Smoke Tests: sidebar has no accessibility violations

### ISSUE THREAD

**Issue by bpasero** [microsoft/vscode#296492]
https://github.com/microsoft/vscode/actions/runs/22220122850/job/64273176390?pr=296485

```
1) VSCode Smoke Tests (Web)
       Accessibility
         Workbench
           sidebar has no accessibility violations:
     Error: Accessibility violations found:

[aria-progressbar-name] ARIA progressbar nodes must have an accessible name (serious)
  Help URL: https://dequeuniversity.com/rules/axe/4.11/aria-progressbar-name?application=axeAPI
  Element: #workbench\.parts\.sidebar > .content > .monaco-progress-container[role="progressbar"][aria-valuemin="0"]
    Class: monaco-progress-container active infinite
    HTML: <div class="monaco-progress-container active infinite" role="progressbar" aria-valuemin="0" style=""><div class="progress-bit" style="background-color: var(--vscode-progressBar-background); width: 2%; opacity: 1;"></div></div>
    Issue: Fix any of the following:
  aria-label attribute does not exist or is empty
  aria-labelledby attribute does not exist, references elements that do not exist or references elements that are empty
  Element has no title attribute

Total: 1 violation(s) affecting 1 element(s)
      at Proxy.assertNoAccessibilityViolations (D:\a\_work\vscode\vscode\test\automation\out\playwrightDriver.js:628:19)
      at runNextTicks (node:internal/process/task_queues:65:5)
      at process.processImmediate (node:internal/timers:453:9)
      at process.callbackTrampoline (node:internal/async_hooks:130:17)
      at async Context.<anonymous> (out\areas\accessibility\accessibility.test.js:47:17)
```
